napari/_qt/qt_viewer.py

```diff
@@ -1112,6 +1114,7 @@ def on_mouse_move(self, event):
         event : vispy.event.Event
             The vispy event that triggered this method.
         """
+        self.viewer.brush_circle_overlay.position = list(event.pos)
```

Comment:
This works, but I'm wondering if it's the right place to add this... does the viewer model have access to cursor position at all times? Cause then we could hook up the event in the VispyOverlay like everything else.

Reply:
I couldn't find a better place. I'm also going to change it slightly - to add a check if the cursor is `circle` to avoid sending position to the brush_circle all the time when the brush is not even activated.

Reply:
I'll see if it's possible to move the position updates to ViewerModel